engine/lib/upgrades/2016092301-0.61-profiles_to_annotations-3b68ae209aa07beb.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Elgg 0.61 upgrade 2016092301
+ * profiles_to_annotations
+ *
+ * Migrate profile metadata to annotations
+ */
+
+$names = array_keys(elgg_get_config('profile_fields'));
```

Comment:
just wondering... what if the profile plugin is not enabled at this moment??? the hook for profile fields is not triggered and that will cause some issues when you activate profile again...